modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,40 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.26]]
+== Neo4j 5.26
+
+=== Deprecated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE db OPTIONS { seedCredentials: ...,  seedConfig: ... }
+----
+| The `CREATE DATABASE` options `seedCredentials` and `seedConfig` have been deprecated.
+For seeding from cloud storage, it is recommended to use `CloudSeedProvider` which will read cloud credentials and configuration from standard locations.
+For further information see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/clustering/databases/#_seed_providers[Seed providers].
```

Comment:
This link will need to be updated to `#_cloud_seed_provider[CloudSeedProvider]`

Reply:
Updated

Reply:
Turns out it's `#cloud-seed-provider`